### ISSUE TITLE
Unable to load plugin

### ISSUE THREAD

**Issue by yasseralsamman** [just-boris/less-plugin-glob#18]
I'm getting this error after installing the plugin: 

Unable to load plugin glob please make sure that it is installed under or at the same level as less

tried removing and reinstalling with no luck.
 
I can see the plugin under /usr/lib/node_modules/less-plugin-glob/

Working on ubuntu 16.04 with lessc 3.0.1

**Comment by just-boris** [comment]
How did you install less and this plugin?

As the error message says, they should be installed in the same folder. Make sure that you have installed them both without `-g` flag.

**Comment by yasseralsamman** [comment]
Both are installed globally, and I can see them both  like 
/usr/lib/node_modules/less
/usr/lib/node_modules/less-plugin-glob

should the plugin folder be somewhere else ?

**Comment by Ciantic** [comment]
Yes this plugin does not seem to work with Less3 on Windows at least.

It can't resolve the plugin.
